clusterloader2/pkg/prometheus/manifests/00monitoring-priorityclass.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: monitoring
+value: 1000000
```

Comment:
Won't it collide with: https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/testing/load/daemonset-priorityclass.yaml
?